templates/web/base/icons/chevron-down.html

```diff
@@ -1 +1 @@
-<svg role="presentation" aria-hidden="true" width="[% width || '1rem' %]" height="[% height || '1rem' %]" viewBox="0 0 16 13" xmlns="http://www.w3.org/2000/svg"><path d="M16 6.5V0L8 6.5L0 0V6.5L8 13L16 6.5Z" fill="currentColor"/></svg>
+<svg ole="presentation" aria-hidden="true" width="[% width || '1rem' %]" height="[% height || '1rem' %]" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M1 12V3L12 12L23 3V12L12 21L1 12Z" fill="currentColor"/></svg>
```

Comment:
Sorry, missed this before, this now has two width and height, should the 24 ones be removed?